src/components/common/navigation/NavigationRoutes.tsx

```diff
@@ -293,6 +303,7 @@ export default function NavigationRoutes() {
             getServerMode()
             if (getEnvironmentData) {
                 getAirGapEnvironmentValue()
+                getManifestScanningEnabled()
```

Comment:
Could have combined into one method

Reply:
for Airgap, we only fetch data if we do not have it in local storage